SWLOR.Game.Server/Feature/UsePerkFeat.cs

```diff
@@ -275,7 +275,12 @@ void CompleteActivation(string id, float abilityRecastDelay)
 
             if (GetIsPC(activator) && !Space.IsPlayerInSpaceMode(activator))
             {
+                ApplyEffectToObject(DurationType.Temporary, EffectSlow(), activator, 1.5f);
```

Comment:
What's the purpose of both a slow and an immobilize at the same time?

Reply:
I asked about this as well, if i remember the discussion the slow was to limit APR to 1. And the immobilize was to make it so they cant move.

Reply:
Immobilize keeps them from moving, ensuring kiting can't be done so easily. However, due to the way NWN handles paralysis, it doesn't take effect smoothly, and this has resulted in people stuttering to a stop. The slow smooths this out, making it feel less janky, with the move speed restored to the level it should be at the end. I'll adjust the timing to make sure there's no overlap between slowing and the timer expiring in the case of weird lag related issues.

Reply:
Also a slow alone isn;t sufficient as even while moving at a slow pace it's possible to disrupt an attacker's attacks per round when they're forced to catch up.